integration/spark/integrations/container/pysparkRddToTableStartEvent.json

```diff
@@ -53,25 +53,25 @@
               "type": "integer"
             }
           ],
-          "_schemaURL": "https://openlineage.io/spec/1-0-0/OpenLineage.json#/definitions/SchemaDatasetFacet"
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/SchemaDatasetFacet"
         },
         "dataSource": {
           "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
           "name": "file",
           "uri": "file",
-          "_schemaURL": "https://openlineage.io/spec/1-0-0/OpenLineage.json#/definitions/DatasourceDatasetFacet"
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/DatasourceDatasetFacet"
         }
       },
       "outputFacets": {
         "outputStatistics": {
           "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
           "rowCount": 0,
           "size": 0,
-          "_schemaURL": "https://openlineage.io/spec/1-0-0/OpenLineage.json#/definitions/OutputStatisticsOutputDatasetFacet"
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/OutputStatisticsOutputDatasetFacet"
         }
       }
     }
   ],
   "producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
-  "schemaURL": "https://openlineage.io/spec/1-0-0/OpenLineage.json#/definitions/RunEvent"
+  "schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/RunEvent"
```

Comment:
_Minor:_ Would be nice if the version bumps in the test data was aligned with the schema change version bump.

Reply:
Would be. I don't know how the schema versions are bumped now- manually? Seems like a future task.

Reply:
Yeah, it's currently manual, but we'll want to try and automate the schema version bumps at some point.